src/views/Shopping/Shopping.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import App from '../../App';
+
+beforeEach(() => {
+  render(<App />);
+});
+
+//delete from list
+it('should delete an item from the list', async () => {
```

Comment:
These tests can also be put in a test file for your list component. Then you can just render that component instead of the entire app